metaflow/plugins/argo/argo_live_run.py

```diff
@@ -46,6 +47,10 @@ def __init__(
         self._metaflow_run = None
         self._argo_run_id = None
         self._metaflow_run_id = None
+        self._time_triggered = time.time()  # this makes more sense to set within .trigger,
+        # but this raises its own issues (what if never triggered, and no variable is set?)
+        # however, redesign (AIP 6507) will move things around and fix this issue.
```

Comment:
Agree that the timer should be put in `trigger` function: Timer should be started only when the flow is triggered. If timer is `None` then the flow has not been started, and no run status or results should be expected.

Reply:
I'll move it to trigger function

Reply:
done